cli/compose/schema/data/config_schema_v3.13.json

```diff
@@ -413,6 +413,12 @@
                 "generic_resources": {"$ref": "#/definitions/generic_resources"}
               },
               "additionalProperties": false
+            },
+            "memswap_limit": {
+              "type": "integer"
+            },
+            "mem_swappiness": {
+              "type": "integer"
```

Comment:
This probably needs a new version of the schema; to do so;

- add a commit that adds a straight copy of the existing schema and  update the default version
- then apply this diff to the copy 

Doing it in 2 commits helps making the diff reviewable